server/trillian_log_signer/Dockerfile

```diff
@@ -8,8 +8,11 @@ ENV DB_USER=test \
 ENV HOST=0.0.0.0 \
     HTTP_PORT=8091
 
-ENV SEQUENCER_GUARD_WINDOW=0s \
-    FORCE_MASTER=true
+ENV SEQUENCER_GUARD_WINDOW=0s
```

Comment:
Thanks. Done. 
Interestingly, travis now complains that the pb generated files differ from what is checked in: https://travis-ci.org/google/trillian/jobs/246141769#L380 
seems unrrelated to this change, though.